src/suggest-item.jsx

```diff
@@ -19,6 +19,32 @@ export default class SuggestItem extends React.Component {
   }
 
   /**
+   * Checking if item just became active and scrolling if needed.
+   * @param {Object} nextProps The new properties
+   */
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isActive && !this.props.isActive) {
+      this.scrollIfNeeded();
+    }
+  }
+
+  /**
+   * Scrolling current item to the center of the list if item needs scrolling.
+   * Item is scrolled to the center of the list.
+   */
```

Comment:
I wrote it in description
c/p
_When there is a lot of fixtures and scroll appears, if you use arrow keys to move through the list you can move to the item which is not visible due to scroll. Fixed it by scrolling item to center if item is not visible._